### PR TITLE
Updated Japanese Localization

### PR DESCRIPTION
**Pull request by KAGUYA-1137**
**Updated Localizable.strings in ja.lproj**

### CODE REVIEW

File: ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,117 +1,118 @@
 //
-// Translation by lisiyaki
+// Translation by KAGUYA-1137
```
Comment (L1ghtmann) [inline ja.lproj/Localizable.strings:2]: Please keep all contributor names

File: ja.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,117 +1,118 @@
...
-"Are you sure that you want to restore from %@?" = "このバクアップから本当に戻したいですか %@?";
+"Please confirm that you have adequate free storage before proceeding:" = "続行前に空き容量が不足していないことを確認してください:";
+"Are you sure that you want to restore from %@?" = "このバックアップから本当に復元しますか？ %@?";
 "Yes" = "はい";
 "No" = "いいえ";
 "Please note:" = "これを読んでください:";
-"You have chosen to restore from a developer backup. This backup includes bootstrap packages." = "あなたはデベロッパーバクアップから戻すことを選択しました。このバクアップにはブートストラップパッケージがあります。";
-"Please confirm that you understand this and still wish to proceed with the restore" = "前の読んだことが分かって、まだ戻したいことを確認してください。";
+"You have chosen to restore from a developer backup. This backup includes bootstrap packages." = "あなたは開発者向けのバックアップから復元することを選択しました。このバックアップにはブートストラップパッケージがあります。
+";
```
Comment (L1ghtmann) [inline ja.lproj/Localizable.strings:21]: Please move the semicolon to the end of the line above

**Review by L1ghtmann — CHANGES_REQUESTED** [review]
Thank you so much for your help! Had two minor comments, but otherwise looks great.

**Comment by KAGUYA-1137** [comment]
> Thank you so much for your help! Had two minor comments, but otherwise looks great.

You're welcome!
I changed two things.
Please confirm.

**Comment by L1ghtmann** [comment]
Semicolon change looks good, but for the other one could you add *both* names to the contributor item, please. Thanks again!

File: ja.lproj/Localizable.strings (hunk 1 above)
Comment (KAGUYA-1137) [reply ja.lproj/Localizable.strings]: Added my name below but it failed the check:(

**Comment by KAGUYA-1137** [comment]
> Semicolon change looks good, but for the other one could you add *both* names to the contributor item, please. Thanks again! 

OK, added *both* contributor name.

**Review by L1ghtmann — APPROVED** [review]
Looks great. Thanks again

**Pull request merged**